src/components/Person/Person.jsx

```diff
@@ -1 +1,22 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">My name is {person.name}</h2>
+    {person.age ? (
+      <p className="Person__age">I am {person.age}</p>
+    ) : (null)}
+
+    <p className="Person__partner">
+      {person.isMarried ? (
+        person.partnerName + " is my " + (
+          person.sex === 'f' ? "husband" : "wife"
+        )
+      ) : (
+        `I am not married`
+      )}
```

Comment:
let's move some part of this logic to var to simplify the readability

Reply:
I don't understand, how can I do some var! Where and how? Can you clear it?